src/AvPortalClient.php

```diff
@@ -68,6 +98,10 @@ public function query(array $options = []): ?array {
       $response = [];
     }
 
+    if ($response !== []) {
+      $this->cacheSet($cid, $response, $this->time->getCurrentTime() + $this->config->get('cache_max_age'));
```

Comment:
I suppose here it will be possible and useful to use also cache tags (of `media_avportal.settings` config).

Reply:
Absolutely! Good catch.

Reply:
Very good remark! Added cache tags and related tests.